ec2/spark_ec2.py

```diff
@@ -1153,6 +1154,7 @@ def real_main():
                     cluster_instances=(master_nodes + slave_nodes),
                     cluster_state='terminated'
                 )
+                print "Deleting security groups (this will take some time)..."
```

Comment:
`wait_for_cluster_state()` prints its own status about the cluster state, so it stays clear to the user. This just makes sure the print about security groups stays grouped with the attempts that follow.

Unfortunately, many functions in spark-ec2 print to stdout, so it's not always clear from the code what gets put to screen unless you dig into the individual funtions.